OpenChange/MAPIStoreAppointmentWrapper.m

```diff
@@ -1830,6 +1837,43 @@ - (int) getPidLidAppointmentReplyTime: (void **) data
   return rc;
 }
 
+
+- (int) getPidLidClientIntent: (void **) data
+                     inMemCtx: (TALLOC_CTX *) memCtx
+{
+  uint32_t longValue;
+
+  if (!itipSetup)
+    [self _setupITIPContext];
+
+  longValue = 0x0000;
```

Comment:
Why don't you define this bit masks as constant as you already did with `PidLidMeetingType` in the same OC PR?